configs/_base_/datasets/scannet_seg-3d-20class.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
```

Comment:
Config file should not import packages. The format should be transferable to YAML and JSON formats.

Reply:
Fixed.